config/makeCoaddTempExp_goodSeeing.py

```diff
@@ -1,11 +1,8 @@
-from lsst.ap.pipe import MaxPsfWcsSelectImagesTask
+from lsst.pipe.tasks.selectImages import BestSeeingWcsSelectImagesTask
 
 config.bgSubtracted = True
-
-# changing coaddName from deep requires modifying the obs_ package--wait
-# until after upcoming RFD
-# config.coaddName='goodSeeing'
-
-config.select.retarget(MaxPsfWcsSelectImagesTask)
+#config.coaddName='goodSeeing'
```

Comment:
Please don't commit commented-out code. If somebody wants to change this flag, they can surely do it themselves.